sdcm/cluster.py

```diff
@@ -2152,14 +2152,11 @@ def _update_db_packages(self, new_scylla_bin, node_list):
 
         def update_scylla_packages(node, queue):
             node.log.info('Updating DB packages')
+            node.remoter.run('mkdir /tmp/scylla')
             node.remoter.send_files(new_scylla_bin, '/tmp/scylla', verbose=True)
             node.remoter.run('sudo yum update -y --skip-broken')
-            node.remoter.run('sudo yum install python34-PyYAML -y')
```

Comment:
we are sure all older versions doesn't need this anymore ? i.e. I think I've added this for rolling upgrades to work

Reply:
We have it also as part of the upgrade_test.py where it's called upgrade_node_packages.
I assume it's not needed at least where I changed it.